mex/model/entities/consent.json

```diff
@@ -0,0 +1,109 @@
+{
+    "$comment": "The our case, the 'specified process or activity' is the publication of contact details of a person (e.g. name and email address) in our metadata catalog. ",
+    "$id": "https://mex.rki.de/schema/entities/consent",
+    "$schema": "http://json-schema.org/draft/2020-12/schema",
+    "description": "Consent of the Data Subject for specified process or activity.",
+    "properties": {
+        "hadPrimarySource": {
+            "$ref": "/schema/entities/primary-source#/identifier",
+            "description": "A primary source for a topic refers to something produced by some agent with direct experience and knowledge about the topic, at the time of the topic's study, without benefit from hindsight. Because of the directness of primary sources, they 'speak for themselves' in ways that cannot be captured through the filter of secondary sources. As such, it is important for secondary sources to reference those primary sources from which they were derived, so that their reliability can be investigated. A primary source relation is a particular case of derivation of secondary materials from their primary sources. It is recognized that the determination of primary sources can be up to interpretation, and should be done according to conventions accepted within the application's domain ([PROV-O, 2013-04-30 ](http://www.w3.org/TR/2013/REC-prov-o-20130430/)).",
+            "sameAs": [
+                "http://www.w3.org/ns/prov#hadPrimarySource"
+            ]
+        },
+        "hasConsentStatus": {
+            "$comment": "Used to indcate if personal details are allowed to publish or not.",
+            "anyOf": [
+                {
+                    "$ref": "/schema/entities/concept#/identifier",
+                    "examples": [
+                        "https://mex.rki.de/item/consent-status-1"
+                    ],
+                    "useScheme": "https://mex.rki.de/item/consent-status"
+                },
+                {
+                    "type": "null"
+                }
+            ],
+            "description": "Specifies the state or status of consent.",
+            "sameAs": [
+                "https://w3id.org/dpv#hasConsentStatus"
+            ]
+        },
+        "hasConsentType": {
+            "$comment": "In DPV there is no property for the type of consent. in DPV, the types are subclasses of dpv:Consent. In order to align with our modelling approach, we model types of consent as a concepts (skos:Concept). Since dpv:Consent is also defined as a skos:Concept, this is no conflict, just another way of implementing it.",
```

Comment:
```suggestion
            "$comment": "In DPV there is no property for the type of consent. In DPV, the types are subclasses of dpv:Consent. In order to align with our modelling approach, we model types of consent as a concept (skos:Concept). Since dpv:Consent is also defined as a skos:Concept, this is no conflict, just another way of implementing it.",
```